build.env

```diff
@@ -30,7 +30,7 @@ GOSEC_VERSION=v2.3.0
 HELM_VERSION=v3.1.2
 
 # minikube settings
-MINIKUBE_VERSION=v1.12.1
+MINIKUBE_VERSION=v1.12.3
```

Comment:
Use `rebase:` as prefix for version changes please.

Reply:
sure